ovirt_metrics.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
 
   # HACK: Rails 5 dropped support for Ruby < 2.2.2
   active_gems_version = "< 5" if Gem::Version.new(RUBY_VERSION) <= Gem::Version.new("2.2.2")
-  spec.add_dependency "activerecord", ">= 4.2.3", (active_gems_version || "< 5.2")
-
+  spec.add_dependency "activerecord", ">= 4.2.3", (active_gems_version || "< 5.3")
+  spec.add_dependency 'sqlite3', "~> 1.3.6"
```

Comment:
Is this patch version important?

Reply:
It was failing on the newest sqlite version, this was one I saw on a thread, but I can test against other versions to see where it fails.

Reply:
why do we need sqlite?

Reply:
Rails 5.0 has a hissy fit if it's not in the gemfile.

Reply:
@jrafanie 
```
An error occurred while loading spec_helper.
Failure/Error: ActiveRecord::Base.establish_connection :adapter => "sqlite3", :database => ":memory:"
Gem::LoadError:
  Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).
```
https://travis-ci.org/ManageIQ/ovirt_metrics/jobs/560506017

Reply:
https://github.com/ManageIQ/ovirt_metrics/blob/master/spec/spec_helper.rb#L94